dev/breeze/tests/test_selective_checks.py

```diff
@@ -1340,7 +1334,12 @@ def test_excluded_providers():
     )
     assert_outputs_are_printed(
         {
-            "excluded-providers-as-string": json.dumps({DEFAULT_PYTHON_MAJOR_MINOR_VERSION: ["cloudant"]}),
+            "excluded-providers-as-string": json.dumps(
+                {
+                    "3.9": ["cloudant"],
+                    "3.13": ["apache.beam", "apache.kafka", "edge3", "fab", "ydb"],
```

Comment:
edge3 is failing because of FAB depedency or any further problem? Still had it on my bucket list but lost track of un-bundling edge3 from FAB... actually it is only needed for API auth...